### ISSUE TITLE
Не находит кошельки. Не сохраняет промежуточный итог.

### ISSUE THREAD

**Issue by boodooboo** [phrutis/LostCoins#50]
Привет. Взял несколько кошельков из небольшого интервала приватных ключей. Перевел их в .bin, потом отсортировал.
Запустил поиск. Всего нужно было проверить 1 860 000 000 000 приватных ключей. На скрине видно, что было проверено +2 600 000 000 000 ключей. Искомые кошельки не найдены.
Так же промежуточный итог не сохраняется.
Что-то не так работает или я неправильно что-то сделал?
Спасибо. 
Запуск такой командой:
LostCoins.exe -t 0 -g -i 0 -x 256,256 -f test-sort.bin -r 1 -s bc91b7409ab79486def978d81a67bd6ab7f2a34be8072895cb7eebdd2e3e3ecc -z bc91b7409ab79486def978d81a67bd6ab7f2a34be8072895cb7eed8e3ee1e6cc -d 2 -n 5
![test](https://user-images.githubusercontent.com/165537/161295935-0adf2d47-ec0b-4bf8-a59c-a09db18817cb.png)

**Comment by phrutis** [comment]
Привет. 
Возможно твой адрес не сжатый?
Возможно программа делит диапазон на 65536 потоков (256x256), а карта 1050 не может обрабатывать такое количество потоков и пропускает.
Попробуй указать -x 128,128   
Ещё лучше переходи на более модифицированную программу Rotor-Cuda,

**Comment by boodooboo** [comment]
Благодарю, Вас, маэстро.

**Comment by Doktor1024** [comment]
Yes, the maestro speaks correctly!) the compactness or non-compactness should be the same for addresses - both in the list and in the program search! The same private gives two different addresses, then this address is translated into a bin! If the program generates compressed addresses, and the downloaded list is not compressed, then it will never find anything.

**Comment by phrutis** [comment]
You can use the mode ```-b``` looking for 2 options, at a speed twice as slow.
```--both``` Search both (Uncompressed and Compressed addresses)
